pkg/scheduler/framework/plugins/volumebinding/assume_cache.go

```diff
@@ -171,29 +171,29 @@ func (c *assumeCache) add(obj interface{}) {
 	if objInfo, _ := c.getObjInfo(name); objInfo != nil {
 		newVersion, err := c.getObjVersion(name, obj)
 		if err != nil {
-			klog.Errorf("add: couldn't get object version: %v", err)
+			klog.ErrorS(err, "Add failed: couldn't get object version")
 			return
 		}
 
 		storedVersion, err := c.getObjVersion(name, objInfo.latestObj)
 		if err != nil {
-			klog.Errorf("add: couldn't get stored object version: %v", err)
+			klog.ErrorS(err, "Add failed: couldn't get stored object version")
 			return
 		}
 
 		// Only update object if version is newer.
 		// This is so we don't override assumed objects due to informer resync.
 		if newVersion <= storedVersion {
-			klog.V(10).Infof("Skip adding %v %v to assume cache because version %v is not newer than %v", c.description, name, newVersion, storedVersion)
+			klog.V(10).InfoS("Skip adding description and name to assume cache because version is not newer than storedVersion", "description", c.description, "name", name, "newVersion", newVersion, "storedVersion", storedVersion)
 			return
 		}
 	}
 
 	objInfo := &objInfo{name: name, latestObj: obj, apiObj: obj}
 	if err = c.store.Update(objInfo); err != nil {
-		klog.Warningf("got error when updating stored object : %v", err)
+		klog.InfoS("Error occurred while updating stored object", "err", err)
 	} else {
-		klog.V(10).Infof("Adding %v %v to assume cache: %+v ", c.description, name, obj)
+		klog.V(10).InfoS("Adding description and name to assume cache", "description", c.description, "name", name, "assumeCache", obj)
```

Comment:
```suggestion
		klog.V(10).InfoS("Adding object to assume cache", "description", c.description, "name", name, "assumeCache", obj)
```

Reply:
Thank you very much for your suggestion, I have changed "name" and "objName" to "cacheKey".